Controller/Adminhtml/RemoveWidget/Index.php

```diff
@@ -86,7 +86,7 @@ public function execute()
         $response = $this->resultJsonFactory->create();
         $response->setHeader('Content-type', 'application/json');
 
-        $storeId = filter_var($this->request->getParam('id'), FILTER_SANITIZE_STRING);
+        $storeId = $this->request->getParam('id');
```

Comment:
FILTER_SANITIZE_STRING used for sanitizing.  Instead of complete removal, should be replaced by something else ?

Reply:
fixed by using strip_tags() and htmlspecialchars()